defaults/capabilities.go

```diff
@@ -0,0 +1,160 @@
+package defaults
+
+const (
```

Comment:
instead of typing these as golang strings, could we define a custom type that just wraps strings?
  Can be `cap`/`capability` or similar